addons/l10n_do/data/account.tax.template.xml

```diff
@@ -923,12 +923,12 @@
             ]"/>
         </record>
 
-        <record id="ret_75_tax_nonformal" model="account.tax.template">
+        <record id="ret_100_tax_nonformal" model="account.tax.template">
```

Comment:
Can't change xml_id of a record in stable version. You should keep the xml_id in this version and rename it only in master (in the automatic fw-port or in a different PR).

Reply:
@bfr-o Done!! Thanks for the info.